themes/10up-theme/TAILWIND.md

```diff
@@ -48,19 +48,51 @@ Tailwind allows you to author custom classes that group multiple utilities toget
 }
 ```
 
-If needed, you may include classes using `@apply` along with your own styles and things will output in the order you include them as well as abitrary value support. This allows you to write your styles in the way that makes the most sense while having all options available to you.
+Depending on the use case, you may include classes using `@apply` along with your own styles and things will output in the order you include them as well as abitrary value support. This allows you to write your styles in the way that makes the most sense while having all options available to you.
```

Comment:
"...along with your own styles and rules / properties will output in the order you include them"